.github/workflows/ci.yml

```diff
@@ -19,6 +19,8 @@ jobs:
     - uses: actions/setup-node@v2
       with:
         node-version: '16'
+    - name: Install dependencies
+      run: sudo apt-get install libx11-dev zlib1g-dev libpng-dev libxtst-dev
```

Comment:
Why is needed again?

Reply:
We currently have the very unfortunate situation that local mac os builds on mac os 11 work fine for both archs, but the builds using the same prebuilds in gh actions fail to launch on intel macs. 

This pr is not meant to be merged, but I am using it to trying to get to the bottom of this issue and the provide a proper fix.

Reply:
So the current release if broken on Intel Macs running macOS 11?

Reply:
No, our current release works fine, as our current release predates the prebuild changes. But current master build with GH actions is broken on intel macs.

Downstream from ffmuc.net ran into this when releasing based on current master: https://github.com/freifunkMUC/jitsi-meet-electron/issues/38

Reply:
Gotcha, thanks for looking into this!